src/librustc/mir/visit.rs

```diff
@@ -247,42 +247,30 @@ macro_rules! make_mir_visitor {
 
             fn super_mir(&mut self,
                          mir: & $($mutability)* Mir<'tcx>) {
-                let Mir {
```

Comment:
(I found quite a few missing things at one point, and adopted this explicit style deliberately...)